### ISSUE TITLE
New Version 4.2.8

### ISSUE THREAD

**Issue by Zombzon** [100apps/charles-hacking#31]
All is in title, is it possible to get it?

**Comment by HeIp** [comment]
Yeah , just cracked it bud, expect the OP to pull in the request soon @Zombzon

**Comment by Zombzon** [comment]
Okay, can you give me the script?

**Comment by HeIp** [comment]
@Zombzon https://github.com/HeIp/charles-hacking/tree/master/4.2.8
You can find it there, please support by staring for future updates.
